### ISSUE TITLE
[feature request] for addon_config.mk: escape $ using $$

### ISSUE THREAD

**Issue by tyhenry** [openframeworks/projectGenerator#144]
this would allow for something like:

	ADDON_LIBS = C:\pathtolib\$$(Platform)\libfile.lib

which would be parsed to 
path: `C:\pathtolib\$(Platform)\`
file: `libfile.lib`

this would be handy for VS since it uses the `$(something)` pattern for macros in project properties

**Comment by kritzikratzi** [comment]
maybe your problem is more specific, but in general i think there are already solutions for adding libs. 

* save the lib files in myAddon/libs/somelib/lib/vs/mylib.lib
* in your addon_config.mk, at the end write: 

```
vs:
    ADDON_LIBS += mylibsfolder
```

ofxKinect might also be interesting to take a look at. it has a lot of platform specific libs. https://github.com/openframeworks/openFrameworks/blob/master/addons/ofxKinect/addon_config.mk

**Comment by tyhenry** [comment]
Yep, that's working fine (see this [OF forum thread](https://forum.openframeworks.cc/t/addon-config-mk-how-to-add-directory-for-visual-studio-additional-library-directories/25533))... 

the request is about how to escape a `$` in the addon_config.mk - in order to keep environmental variables from being parsed and replaced by the project generator, instead leaving it up to the IDE.

It would be helpful because Visual Studio has a lot of macros that use the $(variable) format, like $(Platform), which VS translates to `Win32` or `x64` depending on the build target
